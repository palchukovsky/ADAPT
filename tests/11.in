DECLARE testDecl;

SCOPE level1 {

  DECLARE testDecl;

  SCOPE level2 {

    DECLARE testDecl;

    SCOPE level3 { DECLARE testDecl; }

    USING level3;
    ACCESS testDecl;

  }

  USING level2::level3;

  ACCESS testDecl;

  USING ::level1::level2::level3;
  ACCESS testDecl;

}

USING level1::level2::level3;

ACCESS testDecl;
ACCESS level1::level2::level3::testDecl;
