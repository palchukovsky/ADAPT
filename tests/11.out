ERROR 14
ERROR 20
ERROR 23
ERROR 29
